one_fm/templates/pages/job_application.js

```diff
@@ -138,15 +138,15 @@ job_application = Class.extend({
           btn: this,
           callback: function(r){
             frappe.unfreeze();
-            frappe.msgprint(frappe._("Succesfully Submitted your Job Application and our HR team will be responding to you soon."));
+            frappe.msgprint(frappe._("Successfully Submitted your Job Application and our HR team will be responding to you soon."));
```

Comment:
"Successfully submitted your application. Our HR team will be responding to you soon."

Reply:
@pjamsheer can you please provide the 
 sentences used as a list, we will collaboratively correct it.